### ISSUE TITLE
Error while calling calculateCost

### ISSUE THREAD

**Issue by mjhartley** [Ether1Project/ethofs-sdk-web#3]
I'm getting the following error while setting calculateCost
"throw new Error('Properly formatted ethofs options containing hostingContractSize and hostingContractDuration required');"

It's unclear from the documentation what to set.
These are my options  
 ethofsOptions: {
        hostingContractSize: 1 ,
        hostingContractDuration: 100000
    }

**Comment by IedSupo** [comment]
what if you try higher values for hostingContractSize ?

**Comment by mjhartley** [comment]
have tried 100, 1000 and 10000 - same error

**Comment by IedSupo** [comment]
what's the rest of the code? make sure you got it properly formatted:
`const options = {
    ethofsOptions: {
        hostingContractDuration: 100000,
        hostingContractSize: 20000000
    }
};`

**Comment by mjhartley** [comment]
const ethofsSDK = require('@ethofs/sdk');
const ethofs = ethofsSDK();

const fs = require('fs');
const readableStreamForFile = fs.createReadStream('./images/PsychedelicCryptoCashCow.jpg');

const options = { ethofsOptions: { hostingContractSize: 100000, hostingContractDuration: 100000 } };

ethofs.calculateCost(readableStreamForFile, options).then((result) => {
    //handle results here
    console.log(result);
}).catch((err) => {
    //handle error here
    console.log(err);
});

**Comment by mjhartley** [comment]
I'm just trying some of the SDK examples

**Comment by IedSupo** [comment]
do you use your private key to login ? you need a wallet with etho to make uploads that is bound to your private key.
like this:
 `const ethofsSDK = require('@ethofs/sdk');
const ethofs = ethofsSDK('yourETHOPrivateKey');`

before i  forget it. it's also important that you connect like this :
 `const connections = {
    rpc: 'https://rpc.ether1.org',
    gateway: 'https://gateway.ethoprotocol.com'
};
const ethofsSDK = require('@ethofs/sdk');
const ethofs = ethofsSDK('yourETHOPrivateKey', connections);`

**Comment by mjhartley** [comment]
No, I was trying some of the unauthorised functions, wasn't aware I need to login to get a cost calculation. Thanks for your swift help and support.

**Comment by IedSupo** [comment]
not a problem. you can also join the discord and if you want to develop i'm sure someone will send you some etho
